tox.ini

```diff
@@ -11,7 +11,9 @@ envlist = py26,py27,pep8
 setenv = VIRTUAL_ENV={envdir}
 usedevelop = True
 install_command = pip install -U {opts} {packages}
-deps = -r{toxinidir}/requirements.txt
+deps = -egit+https://git.openstack.org/openstack/neutron@465fb37118abfc04790392365f5e1504aff46dce#egg=neutron
```

Comment:
Curious about what this sha is all about - our a10-openstack install pulls stable/icehouse.  I'm guessing we're either wanting to test against the most recent build of neutron or you've chosen this commit because a certain feature-set was enabled there.